cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue

```diff
@@ -10,23 +10,23 @@
       right-0
       z-20
       mt-2
-      w-max
+      w-48
```

Comment:
As per the design on the smaller screen this width is 10 rem, this is now always 12 rem, Can you check once how this is rendered on smaller screens?

Reply:
https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/latest/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/39E482B1-E229-4C49-87A6-572E76E43F6F?collectionId=ba692773-75df-473a-a41c-38a0647809a4&collectionLayerId=0d8e1432-1eb4-4e26-bfb4-1bce57fd0c92&mode=build&selected=2275281464-363A6AA2-67CF-4193-9885-A4B79B5D7CDC

Reply:
I just noticed it actually has different widths on these two mock-ups (12rem and 10rem):
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/33902374/145426401-8a19f1c6-0be4-4800-aca5-5e69658028f4.png">
I'll double check with Mike and cover this in https://github.com/trustbloc/wallet/issues/1302

Reply:
okay